wdio-local.conf.js

```diff
@@ -21,7 +21,7 @@ exports.config = {
     // directory is where your package.json resides, so `wdio` will be called from there.
     //
     specs: [
-        './e2e/wdio/**/*.e2e-spec.ts'
+        './e2e/wdio/**/*.acs-spec.ts'
```

Comment:
maybe leave 'e2e' as the default for the local config and add acs as a commented out option (to help devs)

Reply:
Done.